### DISCUSSION TITLE
GNSS clock prediction

### DISCUSSION THREAD

**Discussion by laplace76** [groops-devs/groops#117] [Q&A] [answered]
Is there any way to predict the gnss clock, after having loaded a rinex clock file generated from a previous estimation? I have seen a time series extrapolation class exists, but it is not clear to me if it is possible to use it to create a polynomial clock prediction (even for short time intervals : one hour, for example). 

I guess the flow should be something like GnssClockRinex2InstrumentClock => (InterpolatorTimeSeries) => GnssClock2ClockRinex

**Comment by tmayerguerr** [comment, marked as answer]
I think, the programm you are loocking for is [InstrumentResample](https://groops-devs.github.io/groops/html/InstrumentResample.html).
regards
Torsten

**Reply by laplace76** [reply]
Yes, it works. Thanks a lot.
